.github/workflows/jira.yml

```diff
@@ -0,0 +1,61 @@
+name: Create issue on Jira
+
+on:
+  issues:
+    types: opened
+
+jobs:
+  jira:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Jira login
+        id: login
+        uses: atlassian/gajira-login@v3.0.1
+        env:
+          JIRA_BASE_URL: ${{ secrets.JIRA_BASE_URL }}
+          JIRA_USER_EMAIL: ${{ secrets.JIRA_USER_EMAIL }}
+          JIRA_API_TOKEN: ${{ secrets.JIRA_API_TOKEN }}
+
+      - name: Create Jira issue
+        id: create_jira_issue
+        uses: atlassian/gajira-create@v3.0.1
+        with:
+          project: DOCS
+          issuetype: Bug
+          summary: |
+            [GH#${{ github.event.issue.number }}] ${{ github.event.issue.title }}
+          description: |
+            ${{ github.event.issue.body }}
+            ----
+            {panel}
+            [Github permalink|${{ github.event.issue.html_url }}]
+            {panel}
+          fields: '{"customfield_10009": "DOCS-162"}'
```

Comment:
I'm thinking that it could be a good idea to somehow tag the issues created automatically so we can then more easily find the ones coming from the codacy/docs and codacy/pulse-user-docs repositories.

A simple way of doing this can be to just add a label to this field:

![image](https://user-images.githubusercontent.com/60105800/210616608-393edfc5-47c1-4df1-83cf-b663e491cd4b.png)

We can test merging the pull request with the following change, and if it works, we can do a similar update on the codacy/docs workflow:

```suggestion
          fields: '{"customfield_10009": "DOCS-162", "labels": "Pulse"}'
```

Reply:
Great suggestion!